### ISSUE TITLE
amscd: mime type mismatch

### ISSUE THREAD

**Issue by bdtc** [mathjax/MathJax#2492]
### Issue Summary

Using amscd results in a Firefox error and MathJax rendering stops.

### Technical details:

Firefox 68.9.0esr

### Supporting information:

```
<!DOCTYPE html>
<html lang="en-US">
<head>
<meta charset="UTF-8" />

<script
         id="MathJax-script"
         src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"
></script>

</head>
<body>

<p>
\begin{equation}
\begin{CD}
A @>a>> B\
@vvbv @vvcv\
C @>d>> D
\end{CD}
\end{equation}
</p>

</body>
</html>
```

Console error message:

`The resource from “https://cdn.jsdelivr.net/npm/mathjax@3/es5/input/tex/extensions/amsCd.js” was blocked due to MIME type (“text/plain”) mismatch (X-Content-Type-Options: nosniff).`

The Learn More link points to:

[(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options?utm_source=mozilla&utm_medium=firefox-console-errors&utm_campaign=default)]

**Comment by dpvc** [comment]
The error is misleading.  It really means that the file is not found.  That is because of a capitalization error (see #2326).  This was fixed in the MathJax-src repository, but not in the MathJax repository (which is what is used to seed the CDN).  It will be taken care of in the next release.  In the meantime, you can use

``` html
<script>
MathJax = {
  loader: {
    source: {
      '[tex]/amsCd': '[tex]/amscd'
    }
  }
};
</script>
```

to work around it.

**Comment by bdtc** [comment]
Works.

If this fix is still present in the script after the next MathJax release when the file name is updated, will everything still work?

**Comment by dpvc** [comment]
This is actually part of v3.1 for backward compatibility.  (The names of the extensions are being normalized to all lower case, and this remapping will take the older camel-case names to the new names.)  So it will continue to work in 3.1 when it is released.

**Comment by dpvc** [comment]
Fixes in v3.1
